internals/webpack/webpack.base.babel.js

```diff
@@ -21,7 +21,8 @@ module.exports = options => ({
     rules: [
       {
         test: /\.jsx?$/, // Transform all .js and .jsx files required somewhere with Babel
-        exclude: /node_modules/,
+        // excluding full node_modules folder except for react-intl related deps, see: https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#webpack
+        exclude: /node_modules\/(?!(react-intl|intl-messageformat|intl-messageformat-parser)\/).*/,
```

Comment:
IMO, your comment should link to the setup part of the docs instead of the upgrade guide:

https://github.com/formatjs/react-intl/blob/master/docs/Getting-Started.md#esm-build

Reply:
Also, did you check if adding those artifacts to the exclude instead of including them specifically mean they do get transpiled correctly? (If you're not sure, I'll find out when checking this upgrade on an IE setup this week.)

Reply:
You are right, changed the link ;)

I checked the output of `npm run build` and the libraries appeared as chunks, transpiled to ES5, but please doublecheck this on IE.

Reply:
tested using windows env, as windows have different file paths maybe exclude should be 
`/node_modules[\\,/](?!(react-intl|intl-messageformat|intl-messageformat-parser)[\\,/]).*/`

Reply:
Thank you, I changed it in the newest commit.